### PR TITLE
fix: Audit issue xml parser

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**

Bump xml-parser to v^4.4.1

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by legobeat — COMMENTED** [review]
https://github.com/MetaMask/metamask-mobile/pull/10467/files#r1696167930

File: yarn.lock [hunk 1]
```diff
@@ -17845,10 +17845,10 @@ fast-url-parser@1.1.3:
   dependencies:
     punycode "^1.3.2"
 
-fast-xml-parser@^4.0.12, fast-xml-parser@^4.3.4:
-  version "4.3.5"
-  resolved "https://registry.yarnpkg.com/fast-xml-parser/-/fast-xml-parser-4.3.5.tgz#e2f2a2ae8377e9c3dc321b151e58f420ca7e5ccc"
-  integrity sha512-sWvP1Pl8H03B8oFJpFR3HE31HUfwtX7Rlf9BNsvdpujD4n7WMhfmu8h9wOV2u+c1k0ZilTADhPqypzx2J690ZQ==
+fast-xml-parser@^4.0.12, fast-xml-parser@^4.3.4, fast-xml-parser@^4.4.1:
```
[the remark this answers is not in the document]
Comment (tommasini) [reply yarn.lock:17848]: This way we will have the same problem, we need to use the v4.4.1 right? should I change it for something like this on the lockfile?

```
fast-xml-parser@^4.4.1:
```

[the remark this answers is not in the document]
Comment (tommasini) [reply yarn.lock:17848]: Actually I'm having the trouble to keep the v4.4.1 removing the resolution

[the remark this answers is not in the document]
Comment (tommasini) [reply yarn.lock:17848]: Yes!!!! Thank you so much @legobeat I was taking some time to understand it!

**Review by Cal-L — APPROVED** [review]
LGTM

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**